app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :likes
+  has_many :comments
+
+  def update_posts_counter
+    post.increment!(:PostsCounter)
+  end
+
+  def five_recent_comments
+    comments.order(created_at: :desc).limit(5)
```

Comment:
- [ ] Also in line 11 there is a typo error(created_at: :desc) please correcte it.

Reply:
Done